WebRoot/css/zm.css

```diff
@@ -3906,15 +3906,43 @@ HTML>BODY .appt-selected .appt_allday_body
 	height:auto;
 }
 
-.ZmPrefZimletListView .DwtListView-Rows {
-	overflow:visible;
+.ZmPrefZimletDetail {
+	@PrefZimletDetail@
 }
 
-.ZmPrefZimletListView .Row {
-	overflow:hidden;
+.ZOptionsItemsListView {
+	@PanelBorder@
```

Comment:
Yes @SatyamKumarOnline , this can me like this:

```
.ZOptionsItemsListView {
   @PrefSectionTable@ /* or whatever name suits well here */
}
```

and the in property file:
```
PrefSectionTable = @PanelBorder@
```

This will allows to add more custom properties to pref table easily in future.

Thanks